deepchem/models/openvino_model.py

```diff
@@ -0,0 +1,206 @@
+import os
+import io
+import sys
+import itertools
+import subprocess
+
+import numpy as np
+
+import tensorflow as tf
+from tensorflow.python.framework.convert_to_constants import convert_variables_to_constants_v2
+
+try:
+  try:
+    # Import for pip package
+    from model_optimizer import mo_tf
+  except:
+    # Import for OpenVINO distribution
+    import mo_tf
+  from openvino.inference_engine import IECore, StatusCode
+  is_available = True
+except:
+  is_available = False
+  pass
+"""
+Class which wraps Intel OpenVINO toolkit for deep learning inference.
+To enable optimization, pass `use_openvino=True` flag when create a model.
+Read more at https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html
```

Comment:
It would be useful to expand this out further. When would I want to use this class? Could you add an Examples section to the docstring with an usage example for the class?

Reply:
Added Examples, thanks!